ch_pipeline/hfb/containers.py

```diff
@@ -10,6 +10,16 @@
 
 from ..core.containers import FreqContainer, RawContainer
 
+# Try to import bitshuffle to set the default compression options
+try:
+    import bitshuffle.h5
+
+    COMPRESSION = bitshuffle.h5.H5FILTER
+    COMPRESSION_OPTS = (0, bitshuffle.h5.H5_COMPRESS_LZ4)
+except ImportError:
+    COMPRESSION = None
+    COMPRESSION_OPTS = None
+
```

Comment:
```suggestion
from draco.core.containers import COMPRESSION, COMPRESSION_OPTS
```

Since we don't actually include `bitshuffle` as a dependency in `ch_pipeline`, and we already import containers from `draco`, we can just import these objects directly as well. It also means that we only need to maintain this import logic in one place.